documentation/staging/content/userguide/managing-domains/domain-events.md

```diff
@@ -22,15 +22,24 @@ This document describes Kubernetes events that the operator generates about reso
 
 The operator generates these event types in a domain namespace, which indicate the following:
```

Comment:
General comment: 

The doc should keep sections for both 3.x (old) behavior and 4.0 (new) behavior because when the 4.0 doc goes live, the plan is that it'll still be used for 3.x purposes. 4.0 mentions should be kept relatively contiguous and marked with wording like "For 4.0 and later", so that they are easily removed for backport (for immediate publication even though 4.0 hasn't been released). 

Sorry about the extra work, but we're trying minimize the overall workload by making it practical to maintain very similar doc for 4.0 and 3.x concurrently without risking making 4.0 material "live".   

If you'd like to hold off making these updates, @rosemarymarano and I are going to discuss the details of this approach later today.

Reply:
Actually, please hold off. The plan may have changed as of this morning, and may yet change again, Robert has asked me to settle this with Rosemary in the meeting I mentioned.

Reply:
Rosemary and I spoke. Please include a section that briefly summarizes "prior to 4.0" behavior, and, if it's not obvious, the changes that occurred between "prior to 4.0" and "4.0 and later". Use your own judgement about whether the pre-4.0 samples you have replaced/deleted are still useful to revive. The reasoning is that the average reader may be looking for 3.x information even after 4.0 is release (because the live/main docs at that point will be 4.0 docs while the old/3.x docs will not be as easy to leverage), or might be interested in understanding exactly what changed.

Reply:
Also, if you have made any improvements that you think apply to 3.x, please duplicate those changes to documentation/3.3/contents, because 3.3/contents are currently the 'live' docs.

Reply:
Added a section for what has been changed in 4.0.

Reply:
LGTM. Thanks!